Assets/Scripts/Common/Constants.cs

```diff
@@ -2,12 +2,17 @@
 {
   public static class Constants
   {
+    public const float c_defaultMovementSpeed = 12f;
+    public const float c_defaultMouseSensitivity = 100f;
```

Comment:
for public consts (especially in Constants class) I would skip the c_ prefix. It should be used for private consts